orange/src/main/java/com/hartwig/hmftools/orange/report/ReportResources.java

```diff
@@ -46,66 +48,81 @@ public final class ReportResources {
     private static final String FONT_REGULAR_PATH = "fonts/nimbus-sans/NimbusSansL-Regular.ttf";
     private static final String FONT_BOLD_PATH = "fonts/nimbus-sans/NimbusSansL-Bold.ttf";
 
+    @NotNull
+    private final PdfFont fontRegular;
+    @NotNull
+    private final PdfFont fontBold;
+
+    private ReportResources(@NotNull PdfFont fontRegular, @NotNull PdfFont fontBold) {
+        this.fontRegular = fontRegular;
+        this.fontBold = fontBold;
+    }
+
+    public static ReportResources create() {
+        return new ReportResources(createFontFromProgram(loadFontProgram(FONT_REGULAR_PATH)),
+                createFontFromProgram(loadFontProgram(FONT_BOLD_PATH)));
+    }
+
     @NotNull
     public static DecimalFormat decimalFormat(@NotNull String format) {
         // To make sure every decimal format uses a dot as separator rather than a comma.
         return new DecimalFormat(format, DecimalFormatSymbols.getInstance(Locale.ENGLISH));
     }
 
-    @NotNull
-    public static PdfFont fontRegular() {
-        // Cannot be created statically as every PDF needs their own private font objects.
-        return createFontFromProgram(loadFontProgram(FONT_REGULAR_PATH));
+    public Cells cells() {
+        return new Cells(this);
     }
 
-    @NotNull
-    public static PdfFont fontBold() {
-        // Cannot be created statically as every PDF needs their own private font objects.
-        return createFontFromProgram(loadFontProgram(FONT_BOLD_PATH));
+    public Tables tables() {
+        return new Tables(this);
+    }
+
+    public PdfFont fontBold() {
```

Comment:
@kduyvesteyn, again replying here for ease of conversation:

> * I would still suggest to retain the IntelliJ annotations. They are quite widely used in hmftools so ORANGE would become inconsistent if we start removing them.

I was trying to retain the annotations as I made my changes, but it looks like I missed adding some `@NotNull` annotations when I added to this file. Is that what you were referring to?